test/isArray_test.js

```diff
@@ -0,0 +1,57 @@
+goog.module('datalayerhelper.helper.testing.isArray_');
+goog.setTestOnly();
+
+const {isArray_} = goog.require('helper');
+
+describe(`The 'isArray_' function of helper`, () => {
+  /**
+   * Ensure that plain.IsPlainObject recognizes value
+   * to be a array iff expected is true.
+   * @param {*} value The value to check
+   * @param {boolean} expected If the value is a plain object.
+   */
+  function assertIsArray(value, expected) {
```

Comment:
One potential suggestion: It may make the code more readable if you define two helper functions, one "assertIsArray" and one "assertIsNotArray", rather than carrying around "true" and "false" throughout as the "expected" parameter. What do you think?

Reply:
Makes sense, I switched it